ci/requirements-py3.13.yml

```
@@ -0,0 +1,28 @@
+name: test_env
+channels:
+    - defaults
+    - conda-forge
+dependencies:
+    - coveralls
+    - cython
+    - ephem
+    - h5py
+    # - numba  # not available for py3.13 as of Dec 2024
+    - numpy >= 1.17.3
+    - pandas >= 1.3.0
+    - pip
+    - pytest
+    - pytest-cov
+    - pytest-mock
+    - requests-mock
+    - pytest-timeout
+    - pytest-rerunfailures
+    - conda-forge::pytest-remotedata  # version in default channel is old
+    - python=3.13
+    - pytz
+    - requests
+    - scipy >= 1.6.0
+    - statsmodels
+    # - pip:
+        # - nrel-pysam>=2.0  # not available for py3.13 as of Dec 2024
+        # - solarfactors  # required shapely<2 isn't available for Python>=3.12
```

```suggestion
    - pip:
        - nrel-pysam>=2.0
        - solarfactors
```

Both of these are available on PyPI now for py 3.13, I believe